### DISCUSSION TITLE
Is it possible to merge all generated swagger documentations into single one?

### DISCUSSION THREAD

**Discussion by thebadicon** [Burgyn/MMLib.SwaggerForOcelot#258] [Q&A]
Hello, 
first of thank You for the plugin, I use it together with ocelot and it's been great so far.

I'd like to ask everyone, if its possible to put generated swagger documentation into a single swagger.json accessiable via single link?
In other words I'd like to see 
![image](https://user-images.githubusercontent.com/35631475/219052432-9ee7257f-0503-4669-8d62-2fce0920bcc6.png)
in a single dropdown option with all endpoints for all services available there.

I would use the result in creating https://github.com/RicoSuter/NSwag client.

Thank You.

**Comment by Burgyn** [comment]
Hi @thebadicon,
There is no built-in solution. 

What comes to mind is: You can create your own endpoint in Ocelot, where you call generate `swagger.json` for all endpoints (services) and aggregate it yourself.
